code/modules/atmospherics/machinery/pipes/layermanifold.dm

```diff
@@ -0,0 +1,129 @@
+/obj/machinery/atmospherics/pipe/layer_manifold
+	name = "pipe-layer manifold"
+	icon = 'icons/obj/atmospherics/pipes/manifold.dmi'
+	icon_state = "manifoldlayer"
+	desc = "A special pipe to bridge pipe layers with."
+	dir = SOUTH
+	initialize_directions = NORTH|SOUTH
+	pipe_flags = PIPING_ALL_LAYER
+	var/list/front_nodes = list()
+	var/list/back_nodes = list()
+	piping_layer = PIPING_LAYER_DEFAULT
+	device_type = 0

```

Comment:
This should be a define, no?

Reply:
@Cyberboss device type afaik tells it how many connections it can have. unary, binary, trinary, quadranary-howeveryouspell it.
this is special and has up to 10 connections so I was a bit cautious with not making it automatic.